demos/blockfactory/color_picker.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Blockly Demos: Block Factory
+ *
+ * Copyright 2018 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Color picker used to select the color of a toolbox category.
```

Comment:
did you mean to add this file in this commit?

Reply:
Ummm...    what file? :eyes:




(removed)